app/templates/components/forms/admin/settings/system/mail-settings.hbs

```diff
@@ -0,0 +1,90 @@
+<h3 class="ui header">
+  {{t 'Mail Settings'}}
+</h3>
+<div class="grouped fields">
+  <div class="field">
+    <label>
+      {{t 'From Email'}}
+    </label>
+    {{input type='text' name='email_from'}}
+  </div>
+  <div class="field">
+    <label>
+      {{t 'From Name'}}
+    </label>
+    {{input type='text' name='email_from_name'}}
+  </div>
+  <div class="ui hidden divider"></div>
+  <div class="field">
+    {{ui-radio label=(t 'SMTP') name='mail_setting' value='smtp' current='smtp' onChange=(action (mut selectedMailOption))}}
+  </div>
+  <div class="ui basic segment">
+    <div class="grouped {{if (eq selectedMailOption 'sendgrid') 'disabled'}} fields">
```

Comment:
They are disabled via css. Which is not right. If you query the input fields via js and check the disabled property, that still shows as enabled.